### PR TITLE
update naming policy

### PR DESCRIPTION
**Pull request by ashleygwilliams**

### CODE REVIEW

File: user-name-disputes.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+   move on to other platforms leaving their user account unused. Some time
+   later, Irina comes along and wants to use the `coolestname` name as it
+   matches what they use elsewhere.
+2. Ricardo creates a personal user account, `nextbiggestthing`. They're
+   not attached to the name, but thought it would be popular someday. 
+   Later, Irina attempts to create an organization with the
+   `nextbiggestthing` scope matching their company name, Next Biggest
+   Thing Co.
+
+Irina's claims to the already uses scopes is debatable, but their course
```
Comment (seldo) [inline user-name-disputes.md]: *already in-use

File: user-name-disputes.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+When humans solve their problems by talking to other humans with
+respect, everyone has the chance to end up feeling good about the
+interaction.
+
+## Exceptions
+
+Some things are not allowed.
+
+?????
```
Comment (seldo) [inline user-name-disputes.md]: 3 and 7 from the other policy.

If your name is a trademark, we will give it up if another party can prove they own the trademark. If multiple parties control conflict trademarks in different jurisdictions they can go to court and figure it out, but it's not our job.

File: package-name-disputes.md [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+
+These are the base assumptions and values by which naming disputes will
+be evaluated by the npm team:
+
+- A package with a given name should be the package that the most
+  people expect it to be, this means:
+
+    - All things being equal, a package should remain the same package
+    - A popular package name will always point to the same package
+    - A popular trademark will point to an associated package
```
Comment (seldo) [inline package-name-disputes.md]: A trademark should belong to the owner of that trademark, but we require the owner of the trademark to provide proof that they own it.

File: package-name-disputes.md [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+they are brought to the attention of the npm registry admins,
+including but not limited to:
+
+1. Malware (that is, a package designed to exploit or harm the machine
+   on which it is installed).
+2. Violations of copyright or licenses (for example, cloning an
+   MIT-licensed program, and then removing or changing the copyright
+   and license statement).
+3. Illegal content.
+4. "Squatting" on a package name that you *plan* to use, but aren't
```
Comment (seldo) [inline package-name-disputes.md]: 4b. "Typosquatting", i.e. a *deliberately* confusing package meant to pick up attempts to install another package. If two very different packages have similar names that's okay.

File: package-name-disputes.md (hunk 3 above)
Comment (seldo) [inline package-name-disputes.md]: Also we should use the word "registered trademark"

File: package-name-disputes.md [hunk 5]
```diff
@@ -0,0 +1,131 @@
+# Package Names: Dispute Resolution
+
+This document describes the steps that you should take to
+resolve module name disputes with other npm publishers.
+
+This document is a clarification of the acceptable behavior outlined in
+the [npm Code of Conduct](https://www.npmjs.com/policies/conduct), and
+nothing in this document should be interpreted to contradict any aspect
+of the npm Code of Conduct.
```
Comment (seldo) [inline package-name-disputes.md]: "or terms of service"

File: user-name-disputes.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
+# User Names: Dispute Resolution
+
+This document describes the steps that you should take to resolve user name
+disputes with other registered npm users.
+
+This document is a clarification of the acceptable behavior outlined in
+the [npm Code of Conduct](https://www.npmjs.com/policies/conduct), and
+nothing in this document should be interpreted to contradict any aspect
+of the npm Code of Conduct.
```
Comment (seldo) [inline user-name-disputes.md]: "or terms of service"

**Comment by kemitchell** [comment]
@seldo, was there ever a LGTM from you on this PR? We've expanded our existing `disputes.md` somewhat, with specific reference to names, since.

**Comment by LooTKinGz** [comment]
Update files

**Comment by MylesBorins** [comment]
This repository is now archived and the policies now live in https://github.com/npm/documentation/tree/main/content/policies

Closing this PR

**Pull request closed without merging**